supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -343,6 +343,21 @@ Examples:
 
 *See also*: xref:ansible-play[Ansible play]
 
+[[ansible-rulebook]]
+==== image:images/yes.png[yes] Ansible Rulebook (noun)
+*Description*: An _Ansible Rulebook_ tells Event-Driven Ansible what sources to monitor for an event and what to do when certain conditions are met. Rulebooks are written in YAML, and are used like Ansible Playbooks. Write as shown: uppercase A and uppercase R. When using the term _rulebook_ without the Ansible prefix, use lowercase r.
```

Comment:
```suggestion
*Description*: An _Ansible Rulebook_ tells Event-Driven Ansible what sources to monitor for an event and what to do when certain conditions are met. Rulebooks are written in YAML and are used like Ansible Playbooks. Write as shown: uppercase _A_ and uppercase _R_. When using the term _rulebook_ without the Ansible prefix, use lowercase _r_.
```
- Should `Event-Driven` be capitalized? I can't find it listed as an official product name in our product name list: https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1259317633
- remove unnecessary comma following the subordinating conjunction `and`
- Follow guidelines for references to letters as letters per ISG: https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#letters-as-letters

Reply:
Thanks for the feedback @bburt-rh !

- I found Event-Driven Ansible under the "Components, portals, and tools" tab (row 183) of the sheet you referred to but don't see a specific entry for "Event-Driven" on its own. Should we keep it capitalized when referring to "Event-Driven Ansible"? Happy to change if needed.
- Good catch TY on the comma and the letters as letters guideline.

Reply:
@michellemacrh - Good catch! Didn't see it on the other tab. I was looking only on the main product name tab of the spreadsheet.